BookingsApi/BookingsApi.DAL/BookingsDbContext.cs

```diff
@@ -20,6 +20,8 @@ public BookingsDbContext(DbContextOptions options) : base(options)
         public DbSet<HearingVenue> Venues { get; set; }
         public DbSet<Participant> Participants { get; set; }
         public DbSet<JobHistory> JobHistory { get; set; }
+        public DbSet<LinkedParticipant> LinkedParticipants { get; set; }
```

Comment:
Shouldn't need to expose `LinkedParticipants` here. Any reason it's been added?

Reply:
Thats from a merge from master.. not mine

Reply:
Done